packages/FAQ/docs/creating-a-smart-invoice.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Creating a Smart Invoice
+--- 
+
+## Step 1:  Project Details 
+
+Here you will enter your project’s details in sequential order: 
+
+1. Project Name or ID
+2. Link to Project Agreement
+3. Project Description
+4. Project Start / End / Safety Valve (DATE)
+
+<h4><details>
+<summary> What is Safety Valve and Why is it Important?  </summary>    
+<p>&nbsp;</p>
+
+#### Safety Valve is a built in function that allows the Client to withdraw funds from the contract on the date specified in the date entry field.  If either party goes unresponsive or bails on the project, this is the date at which the client is free to withdrawal all funds deposited into Smart Invoice.
+
+</details></h4>
+
+## Step 2:  Payment Details 
+
+In this step you will enter the following payment details: 
+
+1. Client Address 
+2. Service Provider Address 
+3. Total Payment Due 
+4. Payment Token 
+5. Number of Payments 
+6. Arbitration Provider 
+7. Max Dispute Fee
+8. Terms-of-Service
+
+
+## Step 3: Payment Chunks
+
+1. Enter Payment amount per milestone
+2. Enter Amount up until total amount is paid
+
+## Step 4: Create Invoice 
+ 
+Review that all of the details of your invoice are correct, if they are not you will want to re-enter the correct information in each of the fields you filled out incorrectly. 
+
+###### Disclaimer: If you enter the incorrect “Client Address” or if any of the information you provide in the Smart Invoice form is wrong, you will not be able to edit or change that information later.  This is because every completed Smart Invoice form executes on-chain, meaning it cannot be reversed and is immutable.  Therefore, take extra caution and make sure to double or triple check that the details you enter are correct.  
```

Comment:
I'd rather call this a "note" than a "disclaimer". The latter sounds like this is an issue with the app or some vulnerability we need to disclose, which is not the case.